cmd/kubernetes.go

```diff
@@ -229,26 +229,28 @@ func init() {
 	rootCmd.AddCommand(kubernetesCmd)
 }
 
-func fetchKubeConfigFromProvider(op runtime.KubernetesOperator, id string) (*kubeConfig, error) {
+func fetchKubeConfigFromProvider(op runtime.KubernetesOperator, id string) (*kubeConfig, *time.Time, error) {
```

Comment:
Why do you return `*time.Time`?

Reply:
Are there better alternatives?

Reply:
I recommend `time.Time` :D

Reply:
I don't see any reasons to use a pointer here.

Reply:
The null case of `time.Time` is `time.Time{}`.

Reply:
Same thing with `*kubeConfig`, I don't think of any reasons to return a pointer here.

Reply:
Okay, will look into this. Thanks!

Reply:
Done in 3d5f7d4 and b381b2a